node/src/components/binary_port/metrics.rs

```diff
@@ -11,22 +11,16 @@ const BINARY_PORT_TRY_SPECULATIVE_EXEC_COUNT_NAME: &str = "binary_port_try_specu
 const BINARY_PORT_TRY_SPECULATIVE_EXEC_COUNT_HELP: &str =
     "number of TrySpeculativeExec queries received";
 
-const BINARY_PORT_GET_DB_COUNT_NAME: &str = "binary_port_get_db_count";
-const BINARY_PORT_GET_DB_COUNT_HELP: &str = "number of received Get queries related to DB";
+const BINARY_PORT_GET_RECORD_COUNT_NAME: &str = "binary_port_get_db_count";
```

Comment:
```suggestion
const BINARY_PORT_GET_RECORD_COUNT_NAME: &str = "binary_port_get_record_count";
```

Also, please review names of other metrics.